dbms/src/Storages/Page/V3/Blob/BlobStat.h

```diff
@@ -118,8 +124,30 @@ class BlobStats
 
     using BlobStatPtr = std::shared_ptr<BlobStat>;
 
+    class BlobFileIdManager
+    {
+    public:
+        explicit BlobFileIdManager(const PageTypes & page_types);
+
+        // Used at restart to add old blob file id to manager
+        void addFileId(BlobFileId file_id);
+
+        BlobFileId nextFileId(PageType page_type, const std::lock_guard<std::mutex> &);
+
+#ifndef DBMS_PUBLIC_GTEST
+    private:
+#else
+    public:
+#endif
+        std::unordered_map<PageType, BlobFileId> next_file_id_map;
```

Comment:
I think we can make it simpler, and do not need to store a `next_file_id_map`. I.e. 
```
BlobFileId nextFileId(PageType page_type)
{
    auto step = PageType::TypeCountLimit;
    auto id = cur_max_id + step;
    id = id - (id % step) + page_type;
    cur_max_id = id;
    return id;
}
```

Reply:
Fixed.